src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7164,6 +7174,7 @@ public sealed partial class SerializationInfo
         [System.CLSCompliantAttribute(false)]
         public SerializationInfo(System.Type type, System.Runtime.Serialization.IFormatterConverter converter, bool requireSameTokenInPartialTrust) { }
         public string AssemblyName { get { throw null; } set { } }
+        public static bool DeserializationInProgress { get { throw null; } }
```

Comment:
So... we decided to make all of these APIs public?  I thought we were holding off on that until we had 3rd parties signed up to use it.  Did that happen?

Reply:
That's not totally closed. I've removed everything except DeserializationBlockedException, which I believe we agreed could stay so users have something to catch if they want.